docs/source/release/v5.1.0/reflectometry.rst

```diff
@@ -39,6 +39,7 @@ Bug fixes
 - The history for the :ref:`algm-ReflectometryReductionOne` algorithm has been fixed so that the conversion
   to Q is now always included in the history.
 - The sort function in the :ref:`LRDirectBeamSort <algm-LRDirectBeamSort>` algorithm has been updated for Python3.
+- Fixed an issue where `import CaChannel` on Linux would cause a hard crash.
```

Comment:
Generally we would create a new set of release notes for 5.1.1. I should have created those for this branch. I can do that if you wouldn't mind just taking this out of 5.1.0.